### PR TITLE
Tech 6263 migrate exceptional synchrony to GitHub actions

### PR DESCRIPTION
**Pull request by yinonrousso**
## Migrate Exceptional Synchrony from Jenkins to Actions
- add Github workflow
- remove Jenkins files
- remove Jenkins project

### CODE REVIEW

File: .github/workflows/pipeline.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+name: Pipeline
+on: [push]
+jobs:
+  tests:
+    name: Unit Tests
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [2.5, 2.6]
```
Comment (irismoini) [inline .github/workflows/pipeline.yml:11]: I'm a little lost as to why we are running the job for ruby version 2.5 and 2.6?

**Review by irismoini — APPROVED** [review]

File: .github/workflows/pipeline.yml (hunk 1 above)
Comment (yinonrousso) [reply .github/workflows/pipeline.yml:11]: This was just to test against multiple versions of ruby. I'm fine with your suggestion and testing against 2.6.5 as we did with Jenkins

Comment (yinonrousso) [reply .github/workflows/pipeline.yml]: Updated in [d115c98](https://github.com/Invoca/exceptional_synchrony/pull/77/commits/d115c98396e64dec26bec087ab9e659c2ef5d16e)

Comment (irismoini) [reply .github/workflows/pipeline.yml]: nice!

**Pull request merged**